client/client.go

```diff
@@ -100,6 +101,14 @@ func (c *Client) handleEngineEvents() {
 
 // Begin API
 
+// Version returns the go-nitro version
+func (c *Client) Version() string {
+	info, _ := debug.ReadBuildInfo()
+	// TODO this doesn't seem to work, at least locally.
```

Comment:
It looks like this will depend on how we run `main.go`.  From this [comment](https://github.com/golang/go/issues/51831) it turns out if we run specify the file (`go run main.go`) we don't get version info but if we don't (`go run .`) we do get version information.

Although even when we do the version is just `(devel)`:
```
> {"jsonrpc":"2.0","id":1,"method":"version","params":{}}
< {"jsonrpc":"2.0","id":1,"result":"(devel)","error":null}
```

Reply:
For now I've added some logic in https://github.com/statechannels/go-nitro/pull/1200/commits/26df816b4729512b5f1c3e3b9b8e2e6c37e4f1de so that we either get back `(devel)` or the git commit hash (if we built with `-buildvcs`)